src/main/resources/db/changelog/changes/04-create-categories-table.yaml

```
@@ -0,0 +1,30 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-roles-table
+      author: alex
```

```suggestion
      author: alex_liubchenko
```
it's better to pass also last name